--- xfce4-playercontrol-plugin-0.3.0/panel-plugin/playerctrl.h~	2009-01-02 16:24:42.000000000 +0100
+++ xfce4-playercontrol-plugin-0.3.0/panel-plugin/playerctrl.h	2010-11-24 13:06:07.605594781 +0100
@@ -41,7 +41,7 @@
 
 #define CMD_XMMS "xmms"
 #define CMD_BEEP "beep-media-player"
-#define CMD_AUDACIOUS "audacious"
+#define CMD_AUDACIOUS "audacious2"
 #define CMD_AUDACIOUS_OLD "audacious"
 #define CMD_AMAROK "amarok"
 #define CMD_MPD "mpd"
